### PR TITLE
Added theme for maps

### PR DESCRIPTION
**Pull request by erika-tyagi**
Added a `theme_map_behindbars` (similar to [`theme_void`](https://ggplot2.tidyverse.org/reference/ggtheme.html) but keeps the stuff from `theme_behindbars` that we want to keep). Example of theme output below. 

maps-ex

### CODE REVIEW

File: R/theme_behindbars.R [hunk 1]
```diff
@@ -5,9 +5,9 @@
 #' @import ggplot2
 #' @export
 #'
-#' @param base_size base font size, given in pts 
+#' @param base_size base font size, given in pts
 #' @param base_family base font family
-#' 
+#'
 #' @examples
```
Comment (nmmarquez) [inline R/theme_behindbars.R:11]: Can we add an example of how to use the map here for the docs

**Comment by nmmarquez** [comment]
Small change but what do yall think about explicitly loading `ggplot2` in addition to `dplyr`? Now that we have these cool themes and map styles I think it makes sense to have it loaded.

File: R/theme_behindbars.R (hunk 1 above)
Comment (erika-tyagi) [reply R/theme_behindbars.R:11]: Yes, definitely! 

Separate but related: If we (as a team) like the U.S. state hex maps, I can also add a function to barstools that automates making them per our style guide.

Comment (nmmarquez) [reply R/theme_behindbars.R:11]: Love the idea. We should talk about it at our meeting today as well

**Review by hjohns12 — APPROVED** [review]

**Comment by erika-tyagi** [comment]
The facility crosswalks disappeared, so the build will fail, but I think this should be good to go! 
* Added an example for `theme_map_behindbars` 
* Updated the description (depends on `ggplot2` and suggests `usmap` for the example)

**Pull request merged**